internal/dag/httpproxy_processor.go

```diff
@@ -144,6 +144,10 @@ func (p *HTTPProxyProcessor) computeHTTPProxy(proxy *contour_api_v1.HTTPProxy) {
 
 	var tlsEnabled bool
 	if tls := proxy.Spec.VirtualHost.TLS; tls != nil {
+		/*if tls.Passthrough && tls.EnableFallbackCertificate {
+			validCond.AddError(contour_api_v1.ConditionTypeAuthError, "TLSIncompatibleFeatures",
```

Comment:
nit: this should probably end up being `contour_api_v1.ConditionTypeTLSError`

Reply:
Thanks. Fixed